features/id-check-wrapper/internal/src/test/kotlin/uk/gov/onelogin/criorchestrator/features/idcheckwrapper/internal/screen/SyncIdCheckScreenTest.kt

```diff
@@ -68,14 +68,23 @@ class SyncIdCheckScreenTest {
                             biometricTokenResult = biometricTokenResult,
                             delay = readerDelay,
                         ),
-                    configStore = FakeConfigStore(),
```

Comment:
Same comment as below - would it be possible to keep this as-is by updating defaults?